### PR TITLE
update grid axis

### PR DESCRIPTION
**Pull request by Kvaz1r**
Here I add updating for categorical axis and method for separate updating of each of numeric axis. 
Maybe also should rename method Update to UpdateNumerics for explicit specify and solve thus #38.

### CODE REVIEW

**Comment by Kvaz1r** [comment]
@xleclercq , any comments?

**Comment by xleclercq** [comment]
Hi, I have been slow to respond sorry.
For this one please have a look at commit fcc2137e3ecfa8b13881bb75d5dcb316290d306f 
I added an id to the axis: "x" for X-axis and "y" for Y-axis. I gave them a name  because in the future we have to support more than 2 axes.

File: include/wx/charts/wxchartgrid.h [hunk 1]
```diff
@@ -128,6 +128,9 @@ class wxChartGrid : public wxChartElement
     void Shift(double dx,double dy);
     void ChangeCorners(wxDouble minX,wxDouble maxX,
```
Comment (xleclercq) [inline include/wx/charts/wxchartgrid.h]: Let's rename these functions to UpdateAxisLimits to be consistent with the variable names.

Comment (xleclercq) [reply include/wx/charts/wxchartgrid.h]: Also as I mentioned in the comment, I named each axis. So let's not update both axes at the same time but instead have:
void UpdateAxisLimit(const std::string& axisId, wxDouble minX,wxDouble maxX)

File: include/wx/charts/wxchartgrid.h [hunk 2]
```diff
@@ -128,6 +128,9 @@ class wxChartGrid : public wxChartElement
...
         wxDouble minY,wxDouble maxY);
+    void ChangeCorners(wxDouble min,wxDouble max,bool is_X = true);
```
Comment (xleclercq) [inline include/wx/charts/wxchartgrid.h]: If we change the existing ChangeCorners function as explained above then we don't need this one.

File: include/wx/charts/wxchartgrid.h [hunk 3]
```diff
@@ -128,6 +128,9 @@ class wxChartGrid : public wxChartElement
...
+    void ChangeLabels(const wxVector<wxString> &labels,
```
Comment (xleclercq) [inline include/wx/charts/wxchartgrid.h]: Modify this to void ChangeLabels(const std::string& axisId, const wxVector<wxString> &label)

**Pull request merged**